business_requirement/models/business.py

```diff
@@ -40,7 +40,7 @@ def _get_default_company(self):
         states={'draft': [('readonly', False)]}
     )
     ref = fields.Char(
-        'Reference',
+        'WBS',
         required=False,
         readonly=True,
```

Comment:
I am not sure as there are some ACL based on status. I dont know if it has an impact so dont modify it here.